src/commands/close.ts

```diff
@@ -0,0 +1,81 @@
+import { PermissionFlagsBits, SlashCommandBuilder } from "discord.js";
+
+import { IssueClose, PullClose } from "../config/PullComments";
+import { PrivilegedCommand } from "../interfaces/PrivilegedCommand";
+import { errorHandler } from "../utils/errorHandler";
+
+export const close: PrivilegedCommand = {
+  requiredPermissions: [PermissionFlagsBits.ModerateMembers],
+  guildOnly: true,
+  data: new SlashCommandBuilder()
+    .setName("close")
+    .setDescription(
+      "Close an issue or pull request under the freeCodeCamp organisation."
+    )
+    .addStringOption((option) =>
+      option
+        .setName("repository")
+        .setDescription(
+          "The URL-friendly name of the repository to comment on. Must be a freeCodeCamp owned repository."
```

Comment:
```suggestion
          "The repository to comment on in freeCodeCamp/<repo> format."
```

Reply:
I worry that this reads like you have to pass `freeCodeCamp/discord-bot` as the repository value, when it should only be `discord-bot`.

Reply:
```suggestion
          "The name of the repository, under freeCodeCamp's GitHub org, to comment on"
```

Oh is it? Seeing the "URL-friendly name" got me confused.